lib/active_model/serializer/collection_serializer.rb

```diff
@@ -73,8 +72,9 @@ def serializers_from_resources
       def serializer_from_resource(resource, serializer_context_class, options)
         serializer_class = options.fetch(:serializer) { serializer_context_class.serializer_for(resource) }
 
-        if serializer_class.nil? # rubocop:disable Style/GuardClause
-          fail NoSerializerError, "No serializer found for resource: #{resource.inspect}"
+        if serializer_class.nil?
+          ActiveModelSerializers.logger.debug "No serializer found for resource: #{resource.inspect}"
```

Comment:
You mean at serializer lookup on rendered resource but on associations?  Or just always debug?

B mobile phone

> On Jun 9, 2016, at 4:09 AM, Lucas Hosseini <notifications@github.com> wrote:
> 
> In lib/active_model/serializer/collection_serializer.rb:
> 
> > @@ -73,8 +72,9 @@ def serializers_from_resources
> >        def serializer_from_resource(resource, serializer_context_class, options)
> >          serializer_class = options.fetch(:serializer) { serializer_context_class.serializer_for(resource) }
> >  
> > -        if serializer_class.nil? # rubocop:disable Style/GuardClause
> > -          fail NoSerializerError, "No serializer found for resource: #{resource.inspect}"
> > +        if serializer_class.nil?
> > +          ActiveModelSerializers.logger.debug "No serializer found for resource: #{resource.inspect}"
> I think it does, although we should also warn when we can't find a serializer for something that is not inside a collection.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>